ILSpy.ReadyToRun/ILSpy.ReadyToRun.csproj

```diff
@@ -0,0 +1,66 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
+
+  <PropertyGroup>
+    <TargetFramework>net472</TargetFramework>
+    <AssemblyName>ILSpy.ReadyToRun.Plugin</AssemblyName>
+    <LangVersion>7.2</LangVersion>
+
+    <GenerateAssemblyInfo>False</GenerateAssemblyInfo>
+
+    <EnableDefaultItems>false</EnableDefaultItems>
+
+    <BaseAddress>6488064</BaseAddress>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
+    <DebugType>full</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+    <CheckForOverflowUnderflow>True</CheckForOverflowUnderflow>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)' == 'Release'">
+    <DebugType>pdbonly</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <OutputPath>..\ILSpy\bin\$(Configuration)\</OutputPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\ICSharpCode.Decompiler\ICSharpCode.Decompiler.csproj">
+      <Private>False</Private>
+    </ProjectReference>
+    <ProjectReference Include="..\ILSpy\ILSpy.csproj">
+      <Private>False</Private>
+    </ProjectReference>
+    <ProjectReference Include="..\SharpTreeView\ICSharpCode.TreeView.csproj">
+      <Private>False</Private>
+    </ProjectReference>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="ILCompiler.Reflection.ReadyToRun">
+      <HintPath>..\..\runtime\artifacts\bin\coreclr\Windows_NT.x64.Debug\ILCompiler.Reflection.ReadyToRun.dll</HintPath>
+    </Reference>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="ReadyToRunLanguage.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Iced" Version="1.4.0" />
```

Comment:
This should be properly attributed, @0xd4d, what would you like us to write in the README?

Reply:
There's should probably be a THIRD-PARTY-NOTICES.txt file with all this info.

Reply:
We have two places for third party (NuGet) acks:

* at runtime in About via https://github.com/icsharpcode/ILSpy/blob/master/ILSpy/README.txt
* in the repo in https://github.com/icsharpcode/ILSpy/blob/master/README.md (section LICENSE)

Reply:
Yes but the copyright notices aren't part of the file, it might be easier to include that in a separate file than in a README.

Reply:
This is the way we do it, simple as that.

Reply:
I'm not saying you can't put all of that in the README.txt file. The MIT license requires the copyright notices to be part of the binaries:

`The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.`

Reply:
Well, @cshung should have simply asked the project members, there was no need to drag you in for a standard question like this.

Reply:
At runtime the "MIT license" in README.txt gets turned into a hyperlink to `doc/license.txt`.
That file has a clarification which license applies to which parts of the project, so that needs to be updated too.